src/types/custom/vault.ts

```diff
@@ -7,7 +7,7 @@ import { ZapOptions } from "./zapper";
 export interface Apy {
   type: string;
   gross_apr: number;
-  net_apy: number;
+  net_apy: number | string;
```

Comment:
Since for all APY properties are of type number, we should rather cast strings to number instead of allowing strings on this property.

Reply:
That's what I was trying to do originally, but some apy overrides are "NEW" so it needs to be a string

Reply:
maybe we can use Apy.type = 'NEW', and set net_apy = 0 ? not sure if it would be ideal in API repo rather than patching here

Reply:
https://github.com/yearn/yearn-meta/pull/52